docs/get-started/architecture/_index.rst

```diff
@@ -28,18 +35,15 @@ so you can spend your time building models instead of managing infrastructure.
 
 Learn more:
 
--  :doc:`Intro to Determined <introduction>`: Conceptual information about Determined including its
-   features and benefits.
--  :doc:`System Architecture <system-architecture>`: Learn about the main components of the
+-  :ref:`Intro to Determined <introduction-determined>`: Conceptual information about Determined
+   including its features and benefits.
+-  :ref:`System Architecture <system-architecture>`: Learn about the main components of the
    Determined system architecture.
--  :doc:`Distributed Training <../model-dev-guide/dtrain/dtrain-introduction>`: A conceptual
-   overview of distributed training with Determined.
+-  :ref:`Distributed Training <distributed-training-index>`: A conceptual overview of distributed
+   training with Determined.
```

Comment:
many of these updates were merged as part of #8100 so hopefully when you rebase they will be pulled in